app/src/test/java/pack/cpserver/db/DbBackendTest.java

```diff
@@ -22,7 +22,8 @@
 import solid.collectors.ToSolidList;
 import solid.stream.Stream;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 @RunWith(RobolectricTestRunner.class)
 @Config(constants = BuildConfig.class)
```

Comment:
У меня два теста. Можно было в tearDown, но я сюда написал на всякий
случай, да.

чт, 11 авг. 2016, 13:55 matreshkin <notifications@github.com>:

> In app/src/test/java/pack/cpserver/db/DbBackendTest.java
> <https://github.com/artemohanjanyan/CPServer/pull/1#discussion_r74402312>:
>
> > @@ -22,7 +22,8 @@
> >  import solid.collectors.ToSolidList;
> >  import solid.stream.Stream;
> >
> > -import static org.junit.Assert.*;
> > +import static org.junit.Assert.assertEquals;
> > +import static org.junit.Assert.assertTrue;
> >
> >  @RunWith(RobolectricTestRunner.class)
> >  @Config(constants = BuildConfig.class)
>
> То есть в setUp у тебя БД не пустая? Или просто на всякий случай вставил?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/artemohanjanyan/CPServer/pull/1/files/44bb865faaf073fb1a2a813f1734a1f2d28fc658#r74402312>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHH7Y8DMCsrlzN60uTh_rUUDwC64-7Auks5qev-9gaJpZM4Jh_RJ>
> .
>